Re-enable skipped tests

DonJayamanne opened the pull request:
These should have been enabled, these seem to have been disabled due to a merge issue.

DavidKutu commented:
It wasn't a mistake, those two tests are still failing sometimes. You can see it in this very CI run 'Interrupt and running cells again should only run the necessary cells' is failing here:

- https://github.com/microsoft/vscode-jupyter/pull/5792/checks?check_run_id=2519816272
- https://github.com/microsoft/vscode-jupyter/pull/5792/checks?check_run_id=2519795070
- https://github.com/microsoft/vscode-jupyter/pull/5792/checks?check_run_id=2519812002
- https://github.com/microsoft/vscode-jupyter/pull/5792/checks?check_run_id=2519791981

DavidKutu commented:
I'm approving because we should fix them, but at least leave the comments so if someone else works on it they can have that clue.

DonJayamanne commented:
> fix them, but at least leave the comments so if someone else works on it they can have that clue.

The comments were not correct. the failure was due to bugs in our code.
And I added logging and didn't realize you reveted my changes (or as mentioend it was a merge issue) & skipped them.
Ideally you should have got a merge conflict as i removed that code.
